mxcubeweb/core/components/user/usermanager.py

```diff
@@ -275,8 +258,10 @@ def login_info(self):
             )
 
             res["selectedProposalID"] = HWR.beamline.session.proposal_id
+        else:
+            raise Exception("Not logged in")
```

Comment:
Maybe raise a more specific exception type? See `mxcubeweb/routes/login.py` line 85 where is is caught.

```suggestion
            raise NotAuthenticated("Not logged in")
```